lnbits/static/js/base.js

```diff
@@ -64,9 +64,29 @@ window.LNbits = {
         callback
       })
     },
+    createAccount: function (name) {
+      return this.request('post', '/api/v1/wallet', null, {
```

Comment:
- the function is called `createAccount` but it creates a wallet

Reply:
you are right! that is abug